doc/source/user/basics.indexing.rst

```diff
@@ -39,8 +43,7 @@ and accepts negative indices for indexing from the end of the array. ::
     >>> x[-2]
     8
 
-Unlike lists and tuples, NumPy arrays support multidimensional indexing
-for multidimensional arrays. That means that it is not necessary to
+It is not necessary to
 separate each dimension's index into its own set of square brackets. ::
 
     >>> x.shape = (2,5) # now x is 2-dimensional
```

Comment:
Maybe scope creep, but could you reformat these examples with spaces after the commas?

Reply:
Also below (I cannot comment directly there), the sentence 
> It must be noted that the returned array is not a copy of the original,
but points to the same values in memory as does the original array.

should have the glossary term "view" worked in (as a link) since that is the official term